docs/notebooks/cold_plasma_tensor_elements.ipynb

```diff
@@ -0,0 +1,293 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Cold Magnetized Plasma Waves Tensor Elements (S, D, P in Stix's notation)\n",
+    "=========================================================================\n",
+    "\n",
+    "This example shows how to calculate the values of the cold plasma tensor\n",
+    "elements for various electromagnetic wave frequencies.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# First, import some basics (and `PlasmaPy`!)\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from astropy import units as u\n",
+    "from plasmapy.formulary import (cold_plasma_permittivity_SDP,\n",
+    "                                cold_plasma_permittivity_LRP)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's define some parameters, such as the magnetic field magnitude,\n",
+    "the plasma species and densities and the frequency band of interest\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "B = 2 * u.T\n",
+    "species = ['e', 'D+']\n",
+    "n = [1e18 * u.m ** -3, 1e18 * u.m ** -3]\n",
+    "\n",
+    "f = np.logspace(start=6, stop=11.3, num=3001)  # 1 MHz to 200 GHz\n",
+    "omega_RF = f * (2 * np.pi) * (u.rad / u.s)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
```

Comment:
> ```suggestion
>      "outputs_hidden": false
>     },
>     "scrolled": true
> ```
> This is a long output, so let there be scrolling!

Nevermind, this works with notebooks but has yet to be added to `nbsphinx` (https://github.com/spatialaudio/nbsphinx/issues/417). Arrg!